Makefile

```diff
@@ -1,8 +1,32 @@
 SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
 ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+DB_DEV = core
+RAFT_DEV = $(HOME)/.chaincore/raft
 
-null:
-	@echo Please specify a make target.
+default: run
```

Comment:
At risk of opening a bikeshed, what would your opposition to having a sensible default be? Building and running cored is one of the most common things I do in an edit/save/compile/test cycle.